README.md

```diff
@@ -173,7 +173,8 @@ Will give you the following error:
 ## What's next for Prism?
 
 - [x] Server Validation
-- [x] Content Negotiation
+- [x] Accept header validation
+- [x] Content header validation
```

Comment:
Oops!

Reply:
Aren't we doing these?

Reply:
I replaced Accept / Content with "Content Negotiation" in #357 because the two things are both part of content negotiation. If you did this intentionally can you revert it, and if you didnt mean to do this... could you look out for other accidental reverts?

Reply:
You were right. Fixed.